pinot-plugins/pinot-input-format/pinot-confluent-avro/pom.xml

```diff
@@ -35,7 +35,7 @@
   <properties>
     <pinot.root>${basedir}/../../..</pinot.root>
     <kafka.lib.version>2.8.1</kafka.lib.version>
-    <confluent.version>5.3.1</confluent.version>
+    <confluent.version>5.5.3</confluent.version>
```

Comment:
Shall we consolidate the confluent version as a property at `pinot-input-format/pom.xml`?

Reply:
I have moved it to root pom.xml. Make sense to keep all versions there.

Reply:
hmm we should not put versions on plugins on root. since they might not be loaded by default?